common/src/main/java/io/homeassistant/companion/android/database/sensor/SensorDao.kt

```diff
@@ -14,10 +14,10 @@ import kotlinx.coroutines.flow.Flow
 @Dao
 interface SensorDao {
 
-    @Query("SELECT * FROM Sensors WHERE id = :id")
+    @Query("SELECT * FROM sensors WHERE id = :id")
```

Comment:
Was this change intentional?

Reply:
Yes to keep everything lowercase consistent as I saw other where rising the right name of the table lowercase